File: angular/src/app/shared/models/ip-forensics-report/report-list/report-view.model.ts
import { SearchReportModel } from "./search-report.model";

export class ReportViewModel{
    public SearchReportModel: SearchReportModel = new SearchReportModel();
    public ReportBaseModels: Array<ReportBaseModel> = new Array<ReportBaseModel>();
}

export class ReportBaseModel{
    public Id:number=0;
    public IpAddress:string="";
    public AbuseConfidenceScore:string="";
    public TotalReports:string="";
    public LastReportedDate:string="";
    public Continent:string="";
    public Country:string="";
    public Region:string="";
    public City:string="";
    public Mobile:string="";
    public Proxy:string="";
    public Hosting:string="";
    public Tor:string="";
     public CreatedDate:string="";

}

File: Domain/Models/IpForensicsReports/ReportList/ReportViewModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain.Models.IpForensicsReports.ReportList
{
    public class ReportViewModel
    {
        public SearchReportModel SearchReportModel { get; set; }
        public List<ReportBaseModel> ReportBaseModels { get; set; }
    }
    public class ReportBaseModel
    {
        public int Id { get; set; }

        public string IpAddress { get; set; }
        public string AbuseConfidenceScore { get; set; }
        public string TotalReports { get; set; }
        public string LastReportedDate { get; set; }

        public string Continent { get; set; }
        public string Country { get; set; }
        public string Region { get; set; }
        public string City { get; set; }

        public string Mobile { get; set; }
        public string Proxy { get; set; }
        public string Hosting { get; set; }
        public string Tor { get; set; }

        public string CreatedDate { get; set; }
    }
}
